### DISCUSSION TITLE
Creating a screen grab with a keypress

### DISCUSSION THREAD

**Discussion by doobes** [elementary/screenshot-tool#287] [Q&A]
Was suggested I post this here....

I'm finding the inbuilt screenshot utility less than useful. Oftentimes I want to grab screen shots showing menu's and such, and that's all but impossible with the screenshot utility.

I've tried the PrtScr thing, and when I press PrtScr, the screen "blinks" as though something is happening, but when I try to paste into GIMP, it tells me there is nothing to paste.

And why doesn't the inbuilt screenshot utility grab the dock? 

I'm also finding that "Copy to Cliboard" is inop.

Any thoughts about this would be greatly appreciated.

chris

**Comment by stsdc** [comment]
Isn't PrntScr save the image on the disk, instead of saving to clipboard?
